### PR TITLE
Update EIP-5069: Replace handbook with charter

### PR DESCRIPTION
**Pull request by SamWilsn**

### CODE REVIEW

File: EIPS/eip-5069.md
Comment (g11tech) [inline EIPS/eip-5069.md:23]:
```suggestion
 - **Publishing Proposals**: Facilitating high quality proposals, including their history and associated discussions available over the long term at no cost.
```

File: EIPS/eip-5069.md
Comment (g11tech) [inline EIPS/eip-5069.md:45]: should we also add to DONOTs:

` - track or evaluate copyrights or prior art or credits`

File: EIPS/eip-5069.md
Comment (g11tech) [inline EIPS/eip-5069.md:123]: if janitor decides what is `rough consensus` in contentious issues, he/she effectively has veto powers?

Comment (bumblefudge) [reply EIPS/eip-5069.md:123]: well by my interpretation, what the janitor has in this document is more like the authority to defer to a supermajority in overriding an individual's veto. If I may pull an analogy from the USian interpretation of montesquieu, the president can only veto an act of congress if congress votes with less than 2/3 majority.

In a case where one or two editors digs in heels and want to keep discussing, but the rest of the editors feel further deliberation won't be helpful, the difference between "full consensus" (i.e. effective unanimity) and rough consensus (which a janitor could declare) would be more like timeboxing consensus and officially declaring a default to supermajority. It's called "rough" consensus, cuz it can get a little rough!

File: EIPS/eip-5069.md
Comment (bumblefudge) [reply EIPS/eip-5069.md:45]:
```suggestion
- **Track Intellectual Property Claims**: Trademarks, copyrights, and patents are the responsibility of document authors and implementers, not the editors of the EIP process.
- **Track or evaluate Prior Art**: While editors may anecdotally comment on prior art in or outside of the EIP and EVM corpus, there is no expectation or assumption that they will evaluate novelty, assist in prior art review, or track contributions or acknowledgements.

```

File: EIPS/eip-5069.md
Comment (bumblefudge) [reply EIPS/eip-5069.md:123]: help making this clearer in the document would be appreciated, tho.  I think it's strategically imperative that the document promise no vetos so that if people *feel* vetoed, they can feel empowered to ask for an explanation of what's happening and how it's not as simple as a veto-- so being crystal clear on failure modes and interventions is important here.

File: EIPS/eip-5069.md
Comment (SamWilsn) [reply EIPS/eip-5069.md:45]: How about we lump it all into one bullet point:

```suggestion
 - **Provide Legal Advice**: Trademarks, copyrights, patents, prior art, and other legal matters are the responsibility of authors and implementers, not EIP Editors. We are not lawyers, and while we may occasionally make comments touching on these areas, we cannot guarantee any measure of correctness.
```

File: EIPS/eip-5069.md
Comment (SamWilsn) [reply EIPS/eip-5069.md:123]: My exact intention here is that the Janitor can determine _when_ rough consensus has been reached (i.e. ending discussion) but cannot determine _what_ the consensus is.

If a Janitor refuses to declare that consensus has been reached, they can be removed.

File: EIPS/eip-5069.md
Comment (g11tech) [reply EIPS/eip-5069.md:45]: i am fine with either of the versions :+1:

File: EIPS/eip-5069.md
Comment (g11tech) [reply EIPS/eip-5069.md:123]: by deciding `when` it determines the `what`

Comment (SamWilsn) [reply EIPS/eip-5069.md:123]: > by deciding `when` it determines the `what`. may be this point needs to be more chiseled out

So you're suggesting the Janitor might declare consensus has been reached before discussion naturally would've ended?

Comment (g11tech) [reply EIPS/eip-5069.md:110]: i guess so, but then he can be removed so i guess i am good with it

**Review by g11tech — DISMISSED** [review]
lgtm

File: EIPS/eip-5069.md
Comment (Pandapip1) [inline EIPS/eip-5069.md:96]:
```suggestion
Any EIP Editor can call for an election for Janitor. The EIP Editor with the most votes once quorum is met is named Janitor until the next election completes. If there is a tie, the EIP Editors with the most votes will participate in a round-robin cryptographically-secure first-to-one rock-paper-scissors tournament using the following algorithm:

For each pair of participants $(p_1, p_2)$:

1. $p_i$ secretly chooses $c_i$ to be "Rock", "Paper", or "Scissors" and a random string nonce $n_i$.
2. $p_i$ reveals the value of the SHA256 hash of the ASCII-encoded string $c_in_i$ (e.g. "Rock1234".
3. $p_i$ waits for the other player to reveal their hash. If the other player takes more than 10 minutes, $p_i$ wins.
4. Each player reveals $c_i$ and $n_i$. If the other player takes more than 10 minutes, $p_i$ wins.
5. "Rock" beats "Scissors", "Paper" beats "Rock", and "Scissors" beats "Paper". In the case of a draw, GOTO 1.

The EIP Editor with the greatest number of wins wins. In the case of a draw, this process is repeated until there is a winner.

Business continues as usual while the election is running.
```

The algorithm used to randomly determine the winner needs to be fair. I propose this one.

File: EIPS/eip-5069.md
Comment (Pandapip1) [inline EIPS/eip-5069.md:100]:
```suggestion
An EIP Editor is added once quorum is met, provided neither an EIP Editor nor the EIP-Editor-to-be objects.
```

File: EIPS/eip-5069.md
Comment (Pandapip1) [inline EIPS/eip-5069.md:104]:
```suggestion
An EIP Editor can be removed by another EIP Editor once quorum is met, provided no other EIP Editor objects. An EIP Editor can remove themself at any time.

If the Janitor ever ceases to be an EIP Editor, an election for a new Janitor begins immediately. A temporary Janitor must be appointed by the former Janitor.
```

File: EIPS/eip-5069.md
Comment (Pandapip1) [inline EIPS/eip-5069.md:57]:
```suggestion
The Janitor's two responsibilities (on top of their EIP Editor duties) are: to determine when rough consensus has been reached on a matter, and determine when/if it is appropriate to re-open an already settled matter.
```

**Review by Pandapip1 — CHANGES_REQUESTED** [review]

File: EIPS/eip-5069.md
Comment (SamWilsn) [reply EIPS/eip-5069.md]: Went with slightly different wording, but I think I captured what you wanted here.

File: EIPS/eip-5069.md
Comment (SamWilsn) [reply EIPS/eip-5069.md]: image

File: EIPS/eip-5069.md
Comment (SamWilsn) [reply EIPS/eip-5069.md]: "can be" is the wrong word choice here. If all of the following are true:

- an Editor posts a call for input on removing an editor,
- quorum is met, and
- no one objects

Then the editor _is_ removed, not _can be_ removed.

**Comment by SamWilsn** [comment]
Two comments from @gcolvin:

> [W]e don't generally need anyone to call consensus unless someone has a blocking objection, and even then we hope to resolve the blocking issue .  Deciding to move on despite the block is a last resort.  When it really is necessary having an Editor make the call can put that person in a difficult position.  I said on the call that if [@poojaranjan] is willing she would be a better neutral arbitrator. [...] These aren't blocking objections for me, but I don't want to approve it as it is.

> [Replace Janitor with Moderator o]r some title other than Janitor. Their duties are not at all like mopping floors and scrubbing sinks.

**Comment by xinbenlv** [comment]
Thanks for the effort!  I am very encouraged that this could reduce a lot of friction and boost productivity of editing!

A few feedback: 
- 1. I am quite a big fan of making it explicit "we don't decide winner", kudos for spell it out even more clearly, @SamWilsn  👍 
- 2. the current version seems to establish a new role "the janitor" and a power "to interpret consensus", which seems an important decision. 🔢 

- 3.

> After thirty days from the call for input, if not all EIP Editors have responded, the quorum is reduced to the Editors that have responded. This deadline may be extended in exceptional situations.

This means the editors need to be able to properly track responses from others. What's our way to ensure such response.
Shall we define a venue / medium to recognize as legitimate response? i.e. How do we avoid the confusion of 
- "oh, I msged this channel"  
- "I spoke on Zoom meeting"
- "oh I responded on that Github issue"
- "oh I msg someone in DM" etc

For example, Apache requires all important conversation occurs in Emails so there is a integrated way to notify everyone. 

What's our preferred practice?

**Review by gcolvin — CHANGES_REQUESTED** [review]

File: EIPS/eip-5069.md
Comment (Pandapip1) [inline EIPS/eip-5069.md:74]: @SamWilsn can we replace this with the actual em dash unicode character?
```suggestion
For decisions that are unlikely to be controversial—especially for decisions affecting a single proposal—an EIP Editor may choose whatever option they deem appropriate in accordance with our mission.
```

File: EIPS/eip-5069.md
Comment (Pandapip1) [inline EIPS/eip-5069.md:44]:
```suggestion
```

**Review by Pandapip1 — COMMENTED** [review]
I feel like there are some roles missing. Here is what I think needs specifying:

* Janitor/Moderator/<namehere>: Determining when consensus has been reached
* Governing Editor: Participates in governance, voting, and elections
* EIP Repo admin: Greater moderation powers; ability to make major configuration changes
* EIP Repo write: Lesser moderation powers; ability to make minor configuration changes
* EIP Editor: Reviews proposals. Is not necessarily a governing editor

There are a few editor-adjacent roles that have powers that also need specifying:

* eipw/eipw-action maintainers (that's currently @SamWilsn)
* eip-review-bot (and any other bots that interact using @eth-bot) maintainers (that's currently me, @Pandapip1)
* Website maintainers (me & @SamWilsn)
* Other CI maintainers (me & @SamWilsn)

**Comment by SamWilsn** [comment]
> This means the editors need to be able to properly track responses from others. What's our way to ensure such response. Shall we define a venue / medium to recognize as legitimate response? i.e. How do we avoid the confusion of [where to post]? What's our preferred practice?

That's a great question @xinbenlv. I've tried to avoid implementation details as much as possible within this document, because I think they'll end up changing quite frequently.

This particular issue falls directly under the Janitor's duty to "determine when rough consensus has been reached", so wherever the Janitor says opinions/votes have to be registered is where they have to be registered. If the Janitor is unreasonable, vote 'em out.

File: EIPS/eip-5069.md
Comment (SamWilsn) [reply EIPS/eip-5069.md:52]: I really do like the term "Janitor" because other terms, like "moderator", "chairperson", "editor-in-chief" all imply some amount of authority.

I like the implication that the editor in this role "takes out the trash" and does other somewhat messy work. Being the Janitor isn't an honour, but an undesirable position to be in.

**Comment by SamWilsn** [comment]
> I feel like there are some roles missing. Here is what I think needs specifying:

I'd rather make an error by under-specifying than by over-specifying.

> * Janitor/Moderator/<namehere>: Determining when consensus has been reached

We've got this one ;)

> * Governing Editor: Participates in governance, voting, and elections

So this would be "EIP Editor" as written today?

> * EIP Repo admin: Greater moderation powers; ability to make major configuration changes
> * EIP Repo write: Lesser moderation powers; ability to make minor configuration changes

Hm... These feel too specific to GitHub and is are maybe an implementation detail? If the Editors with admin rights misbehave, they can be removed as Editors. I'd love to grant "delete an issue" rights but there's something funky about our GitHub organization that prevents it (or so I've heard.)

> * EIP Editor: Reviews proposals. Is not necessarily a governing editor

So this would be someone who counts for approving PRs, but doesn't get to express opinions/votes for governance purposes?

Does the idea of Topic Groups (see fb4a1a08478b115c9a03d1720f08523df62d0a76) work for this? Or are you talking specifically approve PRs that only EIP Editors can approve?

Does the [delegation of powers](https://github.com/ethereum/EIPs/pull/7468/files#diff-06f522b43b0d427b71ec1519769e35f6fcde1c66aad1096ab564b140e1e9cadeR67) paragraph cover this? An Editor could delegate their PR approving powers to anyone they choose.

> There are a few editor-adjacent roles that have powers that also need specifying:
> 
> * eipw/eipw-action maintainers (that's currently @SamWilsn)
> * eip-review-bot (and any other bots that interact using @eth-bot) maintainers (that's currently me, @Pandapip1)
> * Website maintainers (me & @SamWilsn)
> * Other CI maintainers (me & @SamWilsn)

Do these need to be called out in our organizational charter, or is this more like "If you have questions about this bot, talk to @Pandapip1"?

**Comment by Pandapip1** [comment]
I guess a lot of this can be covered by delegation. While that's not ideal, I won't block for it.

Maybe a seperate document (or documents) detailing implementation-specific powers could be useful?

File: EIPS/eip-5069.md
Comment (SamWilsn) [reply EIPS/eip-5069.md]: I went wit "Keeper of Consensus" (or Keeper for short.)

**Review by abcoathup — APPROVED** [review]

**Review by g11tech — DISMISSED** [review]
lgtm

**Review by xinbenlv — APPROVED** [review]

**Review by FELLIX000 — CHANGES_REQUESTED** [review]
Change General Information

**Pull request merged**
